### ISSUE TITLE
JupyterHub extension does not appear

### ISSUE THREAD

**Issue by nextMJ** [trungleduc/jupyterview#54]
I successfully installed the extension via `pip` as stated in README. Then, I restarted the service (and the server afterward when the extension was not working). The extension is listed in the extension list (see below) but it does not appear in JupyterHub GUI. Double clicking one of the sample files from this repo gives an error (see screenshot below). May I ask for help?

```
jupyter-XXX-singleuser@jupyter:~$ jupyter labextension list
JupyterLab v3.6.3
/var/lib/private/XXX/.local/share/jupyter/labextensions
        jupyter-matplotlib v0.11.3 enabled OK
        @jupyter-widgets/jupyterlab-manager v3.1.4 enabled OK (python, jupyterlab_widgets)

/home/jupyter/mambaforge/envs/jupyter/share/jupyter/labextensions
        jupyterlab_pygments v0.2.2 enabled OK (python, jupyterlab_pygments)
        jupyter-threejs v2.4.1 enabled OK (python, pythreejs)
        nbdime-jupyterlab v2.1.1 enabled OK
        jupyter_cadquery v2.0.2 enabled OK (python, jupyter_cadquery_widgets)
        ipyparallel-labextension v8.6.1 enabled OK
        jupyterview v0.5.1 enabled OK (python, jupyterview)
        jupyterlab-datawidgets v7.1.2 enabled OK
        webgui_jupyter_widgets v0.2.5 enabled OK (python, webgui_jupyter_widgets)
        @jupyterlab/git v0.41.0 enabled OK (python, jupyterlab-git)
        @lckr/jupyterlab_variableinspector v3.0.9 enabled OK (python, lckr_jupyterlab_variableinspector)

Other labextensions (built into JupyterLab)
   app dir: /home/jupyter/mambaforge/envs/jupyter/share/jupyter/lab
```

![obrazek](https://user-images.githubusercontent.com/77353838/232695346-4bbd044b-d291-44e4-b374-f8c6b0338747.png)

**Comment by trungleduc** [comment]
Hi, can you open the log console of your browser and take a screenshot of the error message?

**Comment by nextMJ** [comment]
Thank you for your fast response.

![obrazek](https://user-images.githubusercontent.com/77353838/232705524-dca37a51-c9dc-49d1-a60e-f42d8855ef06.png)

-> Cleaned -> double-clicked `piston.vtu` ->

![obrazek](https://user-images.githubusercontent.com/77353838/232705837-fa7774c7-6f23-4314-b446-ab5f3f6c764f.png)

**Comment by trungleduc** [comment]
ok I see, it's because of the recent changes in the real-time collaboration package of JupyterLab. You can downgrade jupyterlab back to 3.4.x to make `jupyterview` work.

**Comment by trungleduc** [comment]
I need to release a patch for Jupyterlab 3.5 and 3.6, but since 4.0 is just around the corner, I will wait for it and update `jupyterview` later

**Comment by nextMJ** [comment]
Sounds reasonable. Thank you very much for your help :-)

Maybe create a new issue with the patch description and close this one since this problem is "solved"?

**Comment by trungleduc** [comment]
I will add an upper bound for JupyterLab version while waiting for the real patch

**Comment by trungleduc** [comment]
closed by #55
